src/bb-library/Box/TwigExtensions.php

```diff
@@ -38,36 +42,43 @@ public function getDi()
-            'gravatar'  => new Twig\TwigFilter('gravatar','twig_gravatar_filter'),
-            'markdown'  => new Twig\TwigFilter('markdown','twig_markdown_filter', array('needs_environment' => true, 'is_safe' => array('html'))),
-            'truncate'  => new Twig\TwigFilter('truncate','twig_truncate_filter', array('needs_environment' => true)),
-            'timeago'   => new Twig\TwigFilter('timeago','twig_timeago_filter'),
-            'daysleft'  => new Twig\TwigFilter('daysleft','twig_daysleft_filter'),
-            'size'      => new Twig\TwigFilter('size', 'twig_size_filter'),
-            'ipcountryname' => new Twig\TwigFilter('ipcountryname', array($this, 'twig_ipcountryname_filter')),
-            'number' => new Twig\TwigFilter('number', 'twig_number_filter'),
-            'period_title' => new Twig\TwigFilter('period_title', 'twig_period_title', array('needs_environment' => true, 'is_safe' => array('html'))),
-            'autolink' => new Twig\TwigFilter('autolink', 'twig_autolink_filter'),
-            'bbmd' => new Twig\TwigFilter('bbmd', 'twig_bbmd_filter', array('needs_environment' => true, 'is_safe' => array('html'))),
-
-            'bb_date' => new Twig\TwigFilter('bb_date', array($this, 'twig_bb_date')),
-            'bb_datetime' => new Twig\TwigFilter('bb_datetime', array($this, 'twig_bb_datetime')),
-
-            'img_tag' => new Twig\TwigFilter('img_tag', 'twig_img_tag', array('needs_environment' => false, 'is_safe' => array('html'))),
-            'script_tag' => new Twig\TwigFilter('script_tag', 'twig_script_tag', array('needs_environment' => false, 'is_safe' => array('html'))),
-            'stylesheet_tag' => new Twig\TwigFilter('stylesheet_tag', 'twig_stylesheet_tag', array('needs_environment' => false, 'is_safe' => array('html'))),
-
-            'mod_asset_url' => new Twig\TwigFilter('mod_asset_url', 'twig_mod_asset_url'),
-            'asset_url' => new Twig\TwigFilter('asset_url', 'twig_asset_url', array('needs_environment' => true, 'is_safe' => array('html'))),
-
-            'money' => new Twig\TwigFilter('money', 'twig_money', array('needs_environment' => true, 'is_safe' => array('html'))),
-            'money_without_currency' => new Twig\TwigFilter('money_without_currency','twig_money_without_currency', array('needs_environment' => true, 'is_safe' => array('html'))),
-            'money_convert' => new Twig\TwigFilter('money_convert', 'twig_money_convert', array('needs_environment' => true, 'is_safe' => array('html'))),
-            'money_convert_without_currency' => new Twig\TwigFilter('money_convert_without_currency', array('needs_environment' => true, 'is_safe' => array('html'))),
-        );
+        return [
+            /**
+             * 'trans' filter rewrite same filter from Symfony\Bridge\Twig\Extension\TranslationExtension
+             * for compatible with outdated php-gettext library.
```

Comment:
Any reason why we didn't just replace it?

Reply:
All including in this PR related to templates.
Translation filter relate to localization - its another chain of problems for other PR.